MAUI/Popup/popup-animations.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: post
+title: Popup Animations in .NET MAUI Popup control | Syncfusion
+description: Learn here all about Popup Animations support in Syncfusion .NET MAUI Popup (SfPopup) control and more.
+platform: MAUI
+control: SfPopup
+documentation: ug
+--- 
+
+# Popup Animations in .NET MAUI Popup (SfPopup)
+
+## Animation Modes
+
+Built-in animations are available in SfPopup, which is applied when the PopupView opens and closes in the screen.
+The SfPopup has different animation modes as listed below:
+
+* `Zoom`
+* `Fade`
+* `SlideOnLeft`
+* `SlideOnRight`
+* `SlideOnTop`
+* `SlideOnBottom`
+* `None`
+
+N> Setting of AnimationMode is same for both `Displaying popup when the SfPopup is set as root view` and `Displaying popup on the go`. 
```

Comment:
N> Setting of the AnimationMode is same for both `Displaying popup when the SfPopup is set as root view` and `Displaying popup on the go.`